### PR TITLE
Add db demo to ci

### LINKED ISSUE #319: Adicionar db-demo/staging_db dentro do CI para usar o pytest
# Objetivos / Objectivos / Purpose

Adicionar db-demo/staging_db dentro do CI para usar o pytest

### PR DESCRIPTION
**Pull request by esloch**
- Closes #319
- Adding tests to CI
- Adding staging for testing

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -3,8 +3,15 @@ license-file = LICENSE
...
 max-line-length = 79
-exclude = 
+exclude =
     settings.py
     local_settings.py
     */migrations/*
-    
\ No newline at end of file
+
+[tool:pytest]
```
Comment (xmnlab) [inline setup.cfg:11]: is pytest used?

File: AlertaDengue/dbf/tests/test_views.py [hunk 2]
```diff
@@ -1,30 +1,30 @@
+import os
 from django.contrib.auth.models import User
 from django.core.files.base import File
 from django.test import TestCase
+import pytest
```
Comment (xmnlab) [inline AlertaDengue/dbf/tests/test_views.py:5]: is pytest used?

File: AlertaDengue/gis/management/commands/generate_mapfiles.py [hunk 3]
```diff
@@ -36,11 +36,7 @@ def generate_mapfiles_meteorological(self, date_start):
             mf.create_files()
 
     def handle(self, *args, **options):
-        str_date_start = input(
-            'Type the initial date of the the raster'
-            + ' files to process (Y-m-d): '
-        )
-
+        str_date_start = None  # TODO: receive str_date_start from command line
```
Comment (xmnlab) [inline AlertaDengue/gis/management/commands/generate_mapfiles.py:39]: could you create an issue for that?

File: AlertaDengue/load.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # coding=utf-8
 from django.contrib.gis.utils.layermapping import LayerMapping
-from dados.models import (
+from ad_dados.models import (
```
Comment (xmnlab) [inline AlertaDengue/load.py:3]: why ad_dados?

File: Makefile [hunk 5]
```diff
@@ -37,7 +38,8 @@ build_staging:
 build_migrate_staging: build_staging
 	$(staging_compose_cmd) run --rm staging_db postgres -V
 	$(staging_compose_cmd) run --rm staging_web python3 manage.py migrate --noinput
-	$(staging_compose_cmd) run --rm staging_web python3 manage.py migrate --database=forecast --noinput
+	#$(staging_compose_cmd) run --rm staging_web python3 manage.py migrate --database=forecast --noinput
```
Comment (xmnlab) [inline Makefile:41]: any reason for this?

File: Makefile [hunk 6]
```diff
@@ -50,9 +52,10 @@ stop_staging:
 	$(staging_compose_cmd) stop
 
 generate_maps_staging: build_migrate_staging
-	$(staging_compose_cmd) run --rm web python3 manage.py sync_geofiles
-	$(staging_compose_cmd) run --rm web python3 manage.py generate_meteorological_raster_cities
-	$(staging_compose_cmd) run --rm web python3 manage.py generate_mapfiles
+	$(staging_compose_cmd) run --rm staging_web python3 manage.py sync_geofiles
+	# $(staging_compose_cmd) run --rm staging_web python3 manage.py generate_meteorological_raster_cities
```
Comment (xmnlab) [inline Makefile:56]: as it is temporary, could you add a comment above with the number of the issue related to that?

File: Makefile [hunk 7]
```diff
@@ -75,4 +78,14 @@ remove_untagged_images:
...
-	$(staging_compose_cmd) run --rm --no-deps staging_web flake8
\ No newline at end of file
+	$(staging_compose_cmd) run --rm --no-deps staging_web flake8
+
+test_staging_web: generate_maps_staging
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh dados
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh dbf
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh gis
+	$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh api
+	#$(staging_compose_cmd) run --no-deps staging_web bash ../docker/test.sh forecast
```
Comment (xmnlab) [inline Makefile:88]: is there a reason for this? if there is a reason, could you open an issue and add a comment with the issue number?

File: AlertaDengue/dbf/tests/conftest.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
```
[the remark this answers is not in the document]
Comment (esloch) [reply AlertaDengue/dbf/tests/conftest.py:1]: No more...

> @esloch , it is almost ready to be merged! thanks!
> I added some comments here.

Thanks @xmnlab,

**Comment by esloch** [comment]
> @esloch , it is almost ready to be merged! thanks!
> I added some comments here.

Thanks @xmnlab.

File: AlertaDengue/dbf/tests/conftest.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (esloch) [reply AlertaDengue/dbf/tests/conftest.py:1]: no more, i go add the tox.ini file for use in pytest-django

**Pull request merged**